src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -1927,6 +1927,7 @@ internal ImmutableArray<Diagnostic> GetDiagnostics(CompilationStage stage, bool
                             var syntaxTree = syntaxTrees[i];
                             AppendLoadDirectiveDiagnostics(builder, _syntaxAndDeclarations, syntaxTree);
                             builder.AddRange(syntaxTree.GetDiagnostics(cancellationToken));
+                            builder.AddRange(syntaxTree.Options.Errors);
```

Comment:
Let's discuss offline.

---
In reply to: [101836220](https://github.com/dotnet/roslyn/pull/17113#discussion_r101836220) [](ancestors = 101836220)